app/main.py

```diff
@@ -1 +1,61 @@
-#  write your code here
+class Animal:
+
```

Comment:
Remove the blank line between the class definition and the `__init__` method it  follows common Python coding conventions

Reply:
Fix in all places